apps-api/backoffice/src/modules/election/services.ts

```diff
@@ -0,0 +1,148 @@
+import { mapRepositoryError } from '@pple-today/api-common/utils'
+import {
+  Election,
+  ElectionEligibleBallot,
+  ElectionEligibleVoter,
+  ElectionType,
+  EligibleVoterType,
+} from '@pple-today/database/prisma'
+import Elysia from 'elysia'
+import { ok } from 'neverthrow'
+
+import { ElectionStatus, ListElectionResponse } from './models'
+import { ElectionRepository, ElectionRepositoryPlugin } from './repostiory'
+
+export class ElectionService {
+  constructor(private readonly electionRepository: ElectionRepository) {}
+
+  private filterOnsiteElection(election: Election): boolean {
```

Comment:
Please change to `filterActiveOnsiteElection` to specify that this method is used for filter only active election